docs/ManualDeploymentGuideScalarDLonAWS.md

```diff
@@ -0,0 +1,231 @@
+
+* You must have the authority to pull `scalar-ledger` and `scalardl-schema-loader` container images.
+* You must configure the database properties in the helm chart custom values file.
+* You must confirm that replica count of the ledger and envoy pods in the `scalardl-custom-values.yaml` file is equal to the number of nodes with `agentpool=scalardl` label.
+
+### Deploy Scalar DL
+
+Following steps will help you to install Scalar DL on EKS:
+
+1. Download the following Scalar DL configuration files and update the database configuration in `scalarLedgerConfiguration` and `schemaLoading` sections
+    * [scalardl-custom-values.yaml](https://raw.githubusercontent.com/scalar-labs/scalar-kubernetes/master/conf/scalardl-custom-values.yaml)
+    * [schema-loading-custom-values.yaml](https://raw.githubusercontent.com/scalar-labs/scalar-kubernetes/master/conf/schema-loading-custom-values.yaml)
+2. Create the docker-registry secret for pulling the Scalar DL images from the GitHub registry
+    
+   ```console
+    $ kubectl create secret docker-registry reg-docker-secrets --docker-server=ghcr.io --docker-username=<github-username> --docker-password=<github-personal-access-token>
+   ```
+
+3. Run the helm commands on the host machine to install Scalar DL on EKS
+    
+   ```console
+    # Add helm charts 
+    $ helm repo add scalar-labs https://scalar-labs.github.io/helm-charts
+    
+    # List the Scalar charts.
+    $ helm search repo scalar-labs
+    
+    # Load Schema for Scalar DL install with a release name `load-schema`
+    $ helm upgrade --install load-schema scalar-labs/schema-loading --namespace default -f schema-loading-custom-values.yaml
+    
+    # Install Scalar DL with a release name `my-release-scalardl`
+    $ helm upgrade --install my-release-scalardl scalar-labs/scalardl --namespace default -f scalardl-custom-values.yaml
+    ```
+
+Note:
+
+* The same commands can be used to upgrade the pods.
+* Release name `my-release-scalardl` can be changed as per your convenience.
+* `helm ls -a` command can be used to list currently installed releases.
+* You should confirm the load-schema deployment has been completed with `kubectl get po -o wide` before installing Scalar DL.
+* Follow the [Maintain the cluster](./Maintaincluster.md) section for more customization.
+
+## Step 5. Monitor the Cluster
+
+It is critical to actively monitor the overall health and performance of a cluster running in production. 
+You can use Container Insights to collect performance metrics and Fluent Bit to collect logs of the EKS cluster.
+This section will help you to configure monitoring and logging for your EKS cluster.
+
+### Recommendations
+
+* Monitoring should be enabled for EKS in production.
+* CloudWatch agent should be configured in the EKS cluster for collecting metrics from pods.
+* Fluent Bit should be configured in the EKS cluster for collecting logs from pods.
+    * Node instance role must have `CreateLogGroup` permission for Fluent bit.
```

Comment:
Do we only need `CreateLogGroup` policy? 

I think we also need `logs:CreateLogStream` and `logs:PutLogEvents`, etc,

refs:
- `arn:aws:iam::aws:policy/CloudWatchAgentServerPolicy`
- https://gruntwork.io/repos/v0.19.6/terraform-aws-eks/modules/eks-cloudwatch-container-logs

Reply:
While testing it was working only with the `CreateLogGroup` policy. Added `CreateLogStream` and `PutLogEvents` to the document.